confgenerator/prometheus.go

```diff
@@ -90,6 +90,7 @@ func (r PrometheusMetrics) Pipelines(ctx context.Context) ([]otel.ReceiverPipeli
 	exp_otlp_exporter := experimentsFromContext(ctx)["otlp_exporter"]
 	metrics_components := []otel.Component{otel.GroupByGMPAttrs_OTTL()}
 	if exp_otlp_exporter {
+		metrics_components = append(metrics_components, otel.MetricUnknownCounter())
```

Comment:
Lets mention that this is a no-op of the source of the metrics here isn't `prometheus` stuff

Reply:
I'm not sure I follow. This is on the prometheus pipeline, so it would only apply to Prometheus metrics.